.github/workflows/ci.yml

```diff
@@ -45,15 +45,15 @@ jobs:
       - name: PHPCompatibility
         uses: pantheon-systems/phpcompatibility-action@v1
         with:
-          test-versions: 7.4-
+          test-versions: 8.2
   build_test:
     strategy:
       matrix:
-        drupal-version: [ 10 ]
+        drupal-version: [ 10, 11 ]
       fail-fast: false
     runs-on: ubuntu-latest
     container:
-      image: quay.io/pantheon-public/build-tools-ci:8.x-php8.1
+      image: quay.io/pantheon-public/build-tools-ci:8.x-php8.2
```

Comment:
Questions for follow on work: What are we using Build Tools for? Are we testing with a BT site, or just using the extra commands like workflow:wait (which is now in core).

Broadly leading to, can we switch to ubuntu-latest/shivammathur/setup-php/terminus-github-actions and drop the quay dep?

Reply:
should I do that here or in a separate PR?

Reply:
Separate PR is fine, here looking for confirmation on the lift.